File: Matlab_code/Functions/Default_Bandwidth.m
function const = Default_Bandwidth(MAXBW, OBJSIZE, USERCOUNT, user_list)
    const = zeros(USERCOUNT, OBJSIZE);
    for k = 1:1:USERCOUNT
        const(k,k)=MAXBW*user_list(k);
    end
    return 
end

File: Python_code/Default_Bandwidth.py
import numpy as np

def default_bandwidth(MAXBW, OBJSIZE, USERCOUNT, user_list):
    const = np.zeros((USERCOUNT, OBJSIZE))
    for k in range(USERCOUNT):
        const[k, k] = MAXBW * user_list[k]
    return const
